tests/src/packages/samples/GreetingTests.java~

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
We have got GreetingTests.java, no need to add GreetingTests.java~.
Remove this file.